### PR TITLE
CIV-14010 CCD PR for Email notification: Defendant LR Claim Paid in Full

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-14010

### Change description ###

CCD changes for Email notification: Defendant LR Claim Paid in Full

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by German-A-Amoros — APPROVED** [review]

File: ccd-definition/FixedLists/MarkClaimPaidConsent-JO-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/FixedLists/MarkClaimPaidConsent-JO-nonprod.json:1]: Shouldn't this file be named as the others? "SettleDiscontinue" instead of "JO"?

Comment (sjamihmcts) [reply ccd-definition/FixedLists/MarkClaimPaidConsent-JO-nonprod.json:1]: Got this change from CIV-13766(Omaira's PR), will pull the latest changes from Omaira's ticket as it is ready for review.

Comment (sjamihmcts) [reply ccd-definition/FixedLists/MarkClaimPaidConsent-JO-nonprod.json:1]: Have added the same comment in Omaira's ticket as per your suggestion/comment.

Comment (Omaira-Melo-Hmcts) [reply ccd-definition/FixedLists/MarkClaimPaidConsent-JO-nonprod.json:1]: You can merge my PR.  Is already fixed

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**